plugins/duo_admin/komand_duo_admin/tasks/monitor_logs/task.py

```diff
@@ -285,7 +330,7 @@ def add_log_type_field(logs: list, value: str) -> list:
 
     @staticmethod
     def sha1(log: dict) -> str:
-        hash_ = sha1()  # nosec B303
+        hash_ = sha1(usedforsecurity=False)  # nosec B303
         for key, value in log.items():
             hash_.update(f"{key}{value}".encode("utf-8"))
         return hash_.hexdigest()
```

Comment:
I think we can just use our own hashing helper method here instead of remaking it